Tests/NIOCoreTests/ByteBufferQUICLengthPrefixTests.swift

```diff
@@ -0,0 +1,173 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIOCore
+import XCTest
+
+final class ByteBufferQUICLengthPrefixTests: XCTestCase {
+    private var buffer = ByteBuffer()
```

Comment:
Did making this global save us any code?

Reply:
good point, probably not
I just shamelessly stole this from `ByteBufferLengthPrefixTests.swift`

Reply:
so technically it does save us one line per test
But im happy to repeat that line in every test if you think that's better

One advantage would be that a badly written test wouldn't fail other tests

Reply:
It doesn't even save us that much, as several tests immediately shadow it with a local variable. I think it's not worth the cost.